tests/modules/asset_groups/resources/utils.py

```diff
@@ -405,9 +405,16 @@ def remove_files(self):
             shutil.rmtree(asset_group_path)
 
     def cleanup(self):
+        from app.modules.asset_groups.tasks import delete_remote
+
         # Restore original state
         if self.asset_group is not None:
-            self.asset_group.delete()
+            # Don't delete the gitlab project by default
+            with mock.patch('app.modules.asset_groups.tasks.delete_remote'):
+                self.asset_group.delete()
```

Comment:
This will still delete the project locally, I thought you were goint to change the test validation (that nothing had been created locally) to allow the creation of these special Asset Groups. It will speed up the tests